src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -613,17 +627,7 @@ void WaveTrackAffordanceControls::StartEditSelectedClipSpeed(
    if (!interval)
       return;
 
-   PitchAndSpeedDialog dlg(
-      ProjectAudioIO::Get(project).IsAudioActive(), *track, *interval,
-      &GetProjectFrame(project));
-
-   if (wxID_OK == dlg.ShowModal())
-   {
-      ProjectHistory::Get(project).PushState(
-         XO("Changed Pitch and Speed"), XO("Changed Pitch and Speed"),
-         UndoPush::CONSOLIDATE);
-      SelectInterval(project, *interval);
```

Comment:
Again, I don't see the need to move the code -- but also I notice, `SelectInterval` is lost.  Intentional?

Reply:
> SelectInterval is lost
It's a tentative removal. The clip still remains selected after the dialog is closed even without. I intended to ask @crsib if he remembered why he added this in 65d23cf019fe465c852a06314981544fc9699dab.

Reply:
That was literally the task to select the clip on right click. @LWinterberg probably has more details.